js/main.js

```diff
@@ -46,6 +38,10 @@ function handleSubmit(event) {
     $h1.textContent = 'New Entry';
     data.editing = null;
   }
+  $img.setAttribute('src', 'images/placeholder-image-square.jpg');
+  data.nextEntryId++;
+  toggleNoEntries();
```

Comment:
These should still be inside your `if` statement. Only the `$form.reset()` was supposed to be moved